scripts/portstat

```diff
@@ -109,7 +109,7 @@ class Portstat(object):
         if speed is None:
             speed = PORT_RATE
         else:
-            speed = int(speed)/1000
+            speed = float(speed)/float(1000)
```

Comment:
can you not change this? I'm not sure if there is requirement to set speed to integer